pkg/services/store/service.go

```diff
@@ -192,12 +221,21 @@ func ProvideService(sql *sqlstore.SQLStore, features featuremgmt.FeatureToggles,
 				ActionFilesWrite:  allowAllPathFilter,
 				ActionFilesDelete: allowAllPathFilter,
 			}
+
+		// Default ADMIN role is read only
```

Comment:
Given that this is grafana admin -- lets default to readonly rather than always hidden